### DISCUSSION TITLE
ListView can't display

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#13428] [Q&A] [answered]
The Control:
```xaml
        <ListView  ItemsSource="{x:Bind Items,Mode=OneWay}">
            <ListView.ItemTemplate>
               <TextBlock Text="{x:Bind }"/>
            </ListView.ItemTemplate>
        </ListView>
```
Data :
```csharp
        publicObservableCollection<Item> Items { set; get; }  ,
```
In Constructor :
```csharp
Items = new ObservableCollection<Item>(data_array);
```
Display Result:
    no Item is displayed, but the Items actually have data . I also tried  to use ListView.Items property in xaml ,it can correctly work.

Android (7.0) and WASM(firefox-latest) meet this error , WinUI is correct

**Comment by jeromelaban** [comment]
Thanks for the report. Could you attach here a repro sample of your code?

**Reply by DJDQfff** [reply]
https://github.com/DJDQfff/ErrorProject.git

**Reply by jeromelaban** [reply, marked as answer]
Thanks for the report. Let's follow up in https://github.com/unoplatform/uno/issues/13433
